testSrc/unit/io/flutter/run/SdkFieldsTest.java

```diff
@@ -21,18 +21,22 @@
   public void shouldReadFieldsFromXml() {
     final Element elt = new Element("test");
     addOption(elt, "filePath", "lib/main.dart");
+    addOption(elt, "programArgs", "--debug");
```

Comment:
What's a more realistic example? (I think --debug" doesn't do anything since it's the default?)